.github/workflows/rotki_ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           documentation_paths: |
             docs
           backend_paths: |
+            .github
```

Comment:
nah nah don't do that. Let's not run the backend tests unless absolutely necessary. You can run them if you need them absolutley and none of these paths are modified by adding the [run all] tag as @kelsos said